src/pages/products/components/product-card/ProductCard.jsx

```diff
@@ -5,7 +5,6 @@ import { useAuth, useCart, useWishlist } from "../../../../contexts";
 import {
   addToCart,
   addToWishlist,
-  deleteFromCart,
   deleteFromWishlist,
   getOfferPrice,
 } from "../../../../utils/api";

```

Comment:
try to import from a global index file or keep absolute path for readability

Reply:
The way I have structured the folders and file is in such a way that all logically related code will be exported from a single location. Exporting everything from a global `index.js` might raise conflicts at scale